### ISSUE TITLE
stars returns undefined!

### ISSUE THREAD

**Issue by DasCalamity** [marcbachmann/npm-module-search#4]
When console logging the module stars it returns undefined (no error, it just displays undefined)! Everything else works fine!

**Comment by marcbachmann** [comment]
Oh yeah, npm doesn't support stars anymore. They just use the github stars to promote packages.
There's one pull request that removes the stars from the readme:  @https://github.com/marcbachmann/npm-module-search/pull/3

Also in the meanwhile `npm search` got fixed, so we could just wrap that... but that might be a bit slower.

Do you like to have stars listed? We would have to add queries the github projects, but that adds additional requests, so I won't add that in here. In such a case you're better off writing a custom indexer based on the changes feed (skimdb) of npm.

**Comment by DasCalamity** [comment]
Yes I would :D! I also did see that stars was not in the index! I relized then that you probably removed it.
